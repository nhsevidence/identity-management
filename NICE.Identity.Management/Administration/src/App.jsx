import React, { Component } from "react";
import { BrowserRouter, Route } from "react-router-dom";
import { Layout } from "./components/Layout";
import { FindUser } from "./Pages/FindUser";

export default class App extends Component {
  render() {
    return (
      <BrowserRouter>
        <Layout>
          <Route exact path="/" component={FindUser} />
          <Route exact path="/roles" component={FindUser} />
          <Route exact path="/realms" component={FindUser} />
        </Layout>
      </BrowserRouter>
    );
  }
}
