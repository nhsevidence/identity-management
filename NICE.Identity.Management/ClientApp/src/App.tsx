import React, { Component, Fragment } from "react";
// import logo from './logo.svg';
import { BrowserRouter, Route, Link } from "react-router-dom";
import { UsersPage } from "./containers/UsersPage";

class App extends Component {
  render() {
    return (
      <BrowserRouter>
        <Fragment>
          <Route path="/" exact component={UsersPage} />
        </Fragment>
      </BrowserRouter>
    );
  }
}

export default App;
