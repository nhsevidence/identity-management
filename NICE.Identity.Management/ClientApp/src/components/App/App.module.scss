//@import '~@nice-digital/nds-core/scss/core.scss';

// .App {
//   @include clearfix;
// }
