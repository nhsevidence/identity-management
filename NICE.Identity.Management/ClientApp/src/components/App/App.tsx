import React from "react";
import logo from "./logo.svg";
import styles from "./App.module.scss";
import { Tag } from "@nice-digital/nds-tag";

const App: React.FC = () => {
	return (
		<div className={styles.App}>
			<header>
				<img src={logo} alt="logo" />
				<p>
					This is a <Tag alpha>tag</Tag> right here.
					<Tag outline flush>
						Things!
					</Tag>
					<span>
						<Tag flush>Hello</Tag>
					</span>
				</p>
			</header>
		</div>
	);
};

export default App;
