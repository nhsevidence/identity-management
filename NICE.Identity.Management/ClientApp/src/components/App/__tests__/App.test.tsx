import React from "react";
import { shallow, mount } from "enzyme";
import toJson from "enzyme-to-json";
import { App } from "../App";

it("should mount without crashing", () => {
	const wrapper = shallow(<App />);
	expect(wrapper).toHaveLength(1);
});
