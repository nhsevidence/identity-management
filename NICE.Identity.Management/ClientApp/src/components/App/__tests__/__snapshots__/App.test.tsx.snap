// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows login sceren if logged out 1`] = `
<h1>
  you need to log in!
</h1>
`;

exports[`shows secret data if logged in 1`] = `
<div
  className="App"
>
  <button
    onClick={[Function]}
  />
  <header>
    <img
      alt="logo"
      src="logo.svg"
    />
    <h1>
      Hello
    </h1>
    <h2>
      There
    </h2>
    <p>
      This is a 
      <Tag
        alpha={true}
      >
        tag
      </Tag>
       right here.
      <Tag
        flush={true}
        outline={true}
      >
        Things!
      </Tag>
      <span>
        <Tag
          flush={true}
        >
          Hello!
        </Tag>
      </span>
    </p>
  </header>
</div>
`;
